src/components/include/utils/threads/thread.h

```diff
@@ -104,54 +104,45 @@ class Thread : public QObject {
 #else
 class Thread {
 #endif
+
+  typedef enum ThreadCommandEnum {
+    kThreadCommandNone,  // must be first
```

Comment:
@LuxoftAKutsan, according google style guide:
"Enumerators (for both scoped and unscoped enums) should be named either like constants or like macros: either kEnumName or ENUM_NAME."
So far both valid.